example/gardener-local/registry-prow/kustomization.yaml

```
@@ -0,0 +1,67 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../registry
+
+patches:
+  - patch: |
+      - op: replace
+        path: /spec/template/spec/containers/0/env
+        value:
+          - name: REGISTRY_PROXY_REMOTEURL
+            value: http://registry-gcr-io.kube-system.svc.cluster.local:5000
+          - name: REGISTRY_HTTP_ADDR
+            value: :5003
```

Would this be possible as well? (same below)

```suggestion
        path: /spec/template/spec/containers/0/env/0/value
        value: http://registry-gcr-io.kube-system.svc.cluster.local:5000
```

This way, we wouldn't repeat ourselves